bitrise.yml

```diff
@@ -422,7 +425,6 @@ workflows:
     - deploy-to-bitrise-io@1.10:
         inputs:
         - deploy_path: artifacts/
-        - is_compress: 'true'
```

Comment:
This would be the only change we need. 
The others above are just to try that this works as expected

Reply:
Looks like this is the solution, as in this build's PR the zip for each local appears in artifacts directly, no artifacts.zip generated